src/main/java/frc/robot/Robot.java

```diff
@@ -100,6 +100,7 @@ public void robotPeriodic() {
   @Override
   public void disabledInit() {
     CommandScheduler.getInstance().cancelAll();
+    m_robotContainer.autoChooserUpdate();
```

Comment:
why can't this be done in the RobotContainer constructor that is called on robotInit() ?

Reply:
it is called in the constructor of RobotContainer so it is called. This is to refreash the auto commands so you can run them without redeploying robot code.